.codegen/service-pkg.go.tmpl

```diff
@@ -0,0 +1,18 @@
+// Databricks SDK for Go APIs
+// 
+//{{range .Services}}{{if .Description}} - [{{.Package.Name}}.{{.PascalName}}API]: {{.FirstSentence}}
+//
+//{{end}}{{end}}
+package service
+
+import (
+{{range .Packages}}"github.com/databricks/databricks-sdk-go/service/{{.Name}}"
+{{end}}
+)
+
+// trick godoc to use it as relative import
+// See: https://pkg.go.dev/golang.org/x/tools/internal/imports#ImportPathToAssumedName
```

Comment:
Can you elaborate why this is needed? I.e. "otherwise we don't get XYZ".

I assume it has to do with an index on the service package?

Reply:
yep https://pkg.go.dev/github.com/databricks/databricks-sdk-go@v0.1.0/service